src/software/ai/hl/stp/play/kickoff_enemy_play_test.cpp

```diff
@@ -35,19 +36,19 @@ TEST_F(KickoffEnemyPlayTest, test_kickoff_enemy_play)
 
     std::vector<ValidationFunction> terminating_validation_functions = {
         [](std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield) {
-            // Three friendly robots in position to shadow enemy robots. Rectangles are
+            // Two friendly robots in position to shadow enemy robots. Rectangles are
             // chosen to be generally in the way of the the front 3 enemy robots and the
             // friendly goal, based on where the enemy robots are initialized in the test.
             Rectangle robotOneShadowingRect(Point(0, 1.5), Point(-0.4, 1.3));
             Rectangle robotFiveShadowingRect(Point(0, -1.5), Point(-0.4, -1.3));
-            // Rectangle robotThreeShadowingRect(Point(-0.49, 0.1), Point(-0.75, -0.1));
+            Rectangle robotThreeShadowingRect(Point(-0.49, 0.1), Point(-1.8, -0.1));
```

Comment:
Can we make this rectangle smaller? Is -0.75 too small?

Reply:
Sure, do you mean by keeping the rectangle the same size but shifted in the negative x-direction? Currently, Robot 3 is at point (-1.60053, 0.0170163).